aexpect/remote.py

```diff
@@ -0,0 +1,1297 @@
+"""
```

Comment:
The license has to be here even though it was not part of the original source and there has to be a note about the original source like we do in avocado, for example here: https://github.com/avocado-framework/avocado/blob/master/avocado/utils/linux_modules.py

Reply:
I am not sure what you mean by the license "has to be here" since it is already added in the LGTM fixes (last) commit. Thanks for the sample link, I have added

```diff
...
# See LICENSE for more details.
+#
+# This code was imported from the avocado-vt project,
+#
+# virttest/remote.py

"""
Functions and classes used for logging into guests and transferring files.
...
```

However, there was no mentioning of the original author in either avocado-vt/virttest/remote.py or the virt-test. How do we find out who is the original author? The above addition is all I know about the original code.

Reply:
As far as I recall the original author for remote.py is Michael Goldish <mgoldish@redhat.com> at the time. I guess some git repo digging could be done, if my historical recollection is not enough :)

Reply:
I traced the module through avocado-vt and then virt-test and I think it starts somewhere in the autotest repo but I prefer to just trust the actual committer at these times ;)

Reply:
Author added to both modules now, resolving this.